single.html

```diff
@@ -3,65 +3,140 @@
+  <link rel="stylesheet" href="css/single_style.css">
+  <title>Greenfield black tea</title>
 </head>
 
 <body>
 
   <header>
     <div class="container">
-      <a href="#" class="header__logo">
-        <img src="img/logo220.png" alt="">
+      <a href="index.html" class="header__logo">
+        <img src="img/logo_desctop.png" alt="">
       </a>
       <div class="header__bottom">
+        <div class="header__burger"><span></span></div>
         <nav>          
-          <a href="#" class="header__button">Green tea</a>
-          <a href="#" class="header__button">Black tea</a>
-          <a href="#" class="header__button">Other</a>
-          <input type="search" placeholder="Search">
+          <a href="index.html" class="header__button" data-text="Home">Home</a>
+          <a href="gallery.html" class="header__button" data-text="Gallery">Gallery</a>
+          <a href="add_tea.html" class="header__button" data-text="Home">Add tea</a>
         </nav>
-        <div class="header__user__menu">
-          <a href="#" class="header__button">Log In</a>
-          <a href="#" class="login__logo">
-              <img src="img/login.png" alt="login">
-          </a>
+        <div class="header__login">
+          <a href="#" class="header__button" id="login">Log In</a>
         </div>
       </div>
     </div>
   </header>
 
+  <div class="popup">
+    <div id="popup-content">
+      <img src="img/login.svg" alt="User">
+      <input type="text" placeholder="Email">
+      <input type="text" placeholder="Username">
+      <input type="text" placeholder="Password">
+      <label for="">Registration complete</label>
+      <div class="popup__buttons">
+        <button id="login-button"><span>log in</span></button>
+        <button id="registration-button"><span>sign up</span></button>
+        <button id="reg-complete"><span>complete</span></button>
+     </div>
+      <div id="close-login">+</div>
+    </div>
+
+    <div id="comment-popup">
+      <div id="new-comment-stars">
+        1-5 stars here
+      </div>
+      <textarea placeholder="Your comment here..." cols="30" rows="5"></textarea>
+      <button id="comment-button"><span>leave comment</span></button>
+      <div id="close-comment">+</div>
+    </div>
+  </div>
+
   <main>
     <div class="container">
+
       <section class="tea__section">
         <div class="main__information">
-          <img>
-          <p class="tea__fullname">Name</p>
-          <p class="tea__type">Type</p>
-          <p class="tea__stats">Country</p>
-          <p class="tea__stats">Rating</p>
-          <p class="tea__stats">Price</p>
+          <div class="tea__image">
+            <img src="img/blacktea.png" alt="spring melody">
+          </div>
+          <div class="tea__info">
+            <div class="tea__name">Greenfield Spring melody</div>
```

Comment:
Header

Reply:
```suggestion
            <h3 class="tea__name">Greenfield Spring melody</h3>
```
I didnt get. Should it be h2-h4 instead of div?

Reply:
Made it h2

Reply:
Right. Page header text should be wrapped with corresponding header tag. Beware of header hierarchy on the page though.